docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -23,78 +23,125 @@ data-type="inline"
+There are two types of controls:
 
-* *Options list* &mdash; Filters content based on one or more specified options. The dropdown menu is dynamically populated with the results of a terms aggregation. 
-For example, use the options list on the sample flight dashboard when you want to filter the data by origin city and destination city.
+* *Options list* &mdash; Adds a dropdown that allows you to filter the data with one or more options that you select.
++
+For example, if you are using the *[Logs] Web Traffic* dashboard from the sample web logs data, you can add an options list for the `machine.os.keyword` field that allows you to display only the logs that generated from `osx` and `ios` operating systems.
++
+[role="screenshot"]
+image::images/dashboard_controlsOptionsList_8.3.0.png[Options list control for the `machine.os.keyword` field with the `osx` and `ios` options selected]
 
-* *Range slider* &mdash; Filters data within a specified range of numbers. The minimum and maximum values are dynamically populated with the results of a 
-min and max aggregation. For example, use the range slider when you want to filter the sample flight dashboard by a specific average ticket price.
+* *Range slider* &mdash; Adds a slider that allows you to filter the data within a specified range of values. 
++
+For example, if you are using the *[Logs] Web Traffic* dashboard from the sample web logs data, you can add a range slider for the `hour_of_day` field that allows you to display only the log data from 9:00AM to 5:00PM.
 +
 [role="screenshot"]
-image::images/dashboard-controls.png[]
+image::images/dashboard_controlsRangeSlider_8.3.0.png[Range slider control for the `hour_of_day` field with a range of `9` to `17` selected]
 
-To create *Controls* panels:
+[float]
+[[create-and-add-controls]]
+==== Create and add controls
+
+To add interactive filters, create controls, then add them to your dashboard.
+
+. Open or create a new dashboard. 
+
+.. If you are adding a *Control* to an existing dashboard, click *Edit*, then click *Add control* on the dashboard.
+
+.. If you are adding a *Control* to a new dashboard, click *Controls* in the dashboard toolbar, then select *Add control*.
+
+. From the *Data view* dropdown, select the data view with the field you want to appear in the *Control*.
+
+. In the *Field* list, select the field with the documents you want to filter.
++
+The *Control type* is automatically applied for the field you selected.
 
-. On the dashboard, click *Select type*, then select *Controls*.
+. In the *Label* field, enter a clear and self-explanatory label.
 
-. Select the control panel type from the dropdown, then click *Add*.
+. Select the *Minimum width*, or select *Expand width to fit available space*.
 
-. Enter the *Control Label*, then select the *{data-source-caps}* and *Field*.
+. If you are creating an *Options list*, specify if you want to *Allow multiple selections in dropdown* and *Run past timeout*.
```

Comment:
Same thing here - is there another doc we could link to for `Run past timeout`? Or perhaps have a note briefly explaining what this means? It definitely wasn't clear to me at first until @ThomThomson explained what exactly this did when I was testing it